src/server/src/test/resources/cassandra-reaper-https-at.yaml

```diff
@@ -0,0 +1,105 @@
+# Copyright 2017-2017 Spotify AB
```

Comment:
Suggestion: Looks like cassandra-reaper-http-at.yaml is no longer used anywhere but still exists. Should we remove it?